### DISCUSSION TITLE
States terminology improvements

### DISCUSSION THREAD

**Discussion by StefanoFalsetto-CKHIOD** [camaraproject/CallForwardingSignal#110] [Ideas]
The terms used into the returned states of conditional call forwarding are not standard. Maybe in next version we can change them using HLR terminology?
My proposal:

- from `conditional_unavailable` to `conditional_not_reachable`
- from `conditional_no_reply` to `conditional_no_answer`

**Comment by FabrizioMoggio** [comment]
Agreed.

**Comment by FabrizioMoggio** [comment]
New Issue to fix what discussed here: https://github.com/camaraproject/CallForwardingSignal/issues/111

**Comment by bigludo7** [comment]
Fine for me as well

**Comment by FabrizioMoggio** [comment]
Implemented
